MSAL/src/requests/MSALInteractiveRequest.m

```diff
@@ -131,8 +133,16 @@ - (NSURL *)authorizationUrl
     if (user)
     {
         parameters[MSID_OAUTH2_LOGIN_HINT] = user.displayableId;
-        parameters[MSID_OAUTH2_LOGIN_REQ] = user.uid;
-        parameters[MSID_OAUTH2_DOMAIN_REQ] = user.utid;
+        
+        if (user.uid)
```

Comment:
nit: there's no difference if uid/utid is nil and we set it

Reply:
changed